crates/figma_import/tests/layout_tests.rs

```diff
@@ -82,48 +82,44 @@ fn add_view_to_layout(
             }
         }
         if use_measure_func {
-            manager
-                .add_style_measure(
-                    my_id,
-                    parent_layout_id,
-                    child_index,
-                    view.style.layout_style.clone(),
-                    view.name.clone(),
-                    measure_func,
-                )
-                .unwrap();
+            let _ = manager.add_style(
```

Comment:
Yes unwrap() is ok. I didn't notice it used to be unwrap(). It might because of the rebase since the original change was made a while ago. I changed them back.